Makefile

```
@@ -6,6 +6,8 @@ modules: vtty.c vtty.h
 
 modules_install: modules
 	$(MAKE) -C $(KDIR) M=$(PWD) modules_install
+	depmod -A
+	install -m 644 50-vtty.rules /etc/udev/rules.d
```

```suggestion
	install -m 644 50-vtty.rules /usr/lib/udev/rules.d
```
This will allow user to override the rule in /etc/udev/ if needed.

---

Done.